packages/@o3r/components/src/tools/placeholder/placeholder.component.ts

```diff
@@ -30,7 +31,11 @@ export class PlaceholderComponent implements OnInit, OnDestroy {
 
   private subscription = new Subscription();
 
-  private id$ = new ReplaySubject<string>(1);
+  private id$ = new BehaviorSubject<string | undefined>(undefined);
+
+  private readonly afterViewInit$ = new ReplaySubject<void>(1);
```

Comment:
Not sure it needs to be `Replay`, I am even not sure it is not causing an issue an always bypass the `sample` operator :S